src/neural/CL/OpenCL.cc

```diff
@@ -53,19 +53,31 @@ const std::string sourceCode_convolve3 =
 #include "clsource/convolve3.opencl"
     ;
 
-// Important: keep the follow includes in the same orders.
-// Beware of using a formatting tool like clang-format
-// that could mess with this order.
-// The correct order is: common/part1/part2/part3/batched.
```

Comment:
This comment is incredibly useful to anyone reading this code for the first time. Perhaps remove the comment about linters, but the part about preserving the correct order is very important and should be kept IMO.

Reply:
I removed it because I found a way around and it was no longer necessary.

Reply:
See the "critical change" in the initial comment of the PR.

Reply:
Now you can clang-format the OpenCL.cc file all you want.

Reply:
Fair enough.

Reply:
You can do
```
// clang-format off
around the code you don't want to be touched by clang-format
// clang-format on
```